content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html

```diff
@@ -0,0 +1,40 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - अपने ऐप को उजागर करना
```

Comment:
```suggestion
title: इंटरएक्टिव ट्यूटोरियल - अपने ऐप को एक्सपोज़ करना
```

Reply:
Do check the sheet [here](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0) for words like these!! 
Some of them may have already been discussed & we encourage to use the same every where